app/controllers/proposal_controller.rb

```diff
@@ -89,9 +90,13 @@ def destroy
       return
     end
 
-    @event.save(validate: false)
-    redirect_to conference_program_proposal_index_path(conference_id: @conference.short_title),
-                notice: 'Proposal was successfully withdrawn.'
+    if @event.save
```

Comment:
Do we really need to validate when we withdraw the event? 
If the event is resubmitted, validations will run then.